CONTRIBUTING.md

```diff
@@ -0,0 +1,109 @@
+# Contributing to SafetyCLI
+
+First off, thanks for taking the time to contribute! We welcome contributions from everyone and are grateful for your help in making SafetyCLI better.
+
+## Table of Contents
+- [Contributing to SafetyCLI](#contributing-to-safetycli)
+  - [Table of Contents](#table-of-contents)
+  - [How Can I Contribute?](#how-can-i-contribute)
+    - [Reporting Bugs](#reporting-bugs)
+    - [Suggesting Enhancements](#suggesting-enhancements)
+    - [Improving Documentation](#improving-documentation)
+  - [Getting Started](#getting-started)
+    - [Code of Conduct](#code-of-conduct)
+    - [Development Setup](#development-setup)
+    - [Running Tests](#running-tests)
+  - [Pull Request Process](#pull-request-process)
+    - [Commit Messages](#commit-messages)
+    - [Merging Changes](#merging-changes)
+  - [License](#license)
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+If you find a bug, please report it by opening a [GitHub issue](https://github.com/pyupio/safety/issues). When reporting a bug, please include:
```

Comment:
Is it worth linking to our bug bounty page?

Reply:
For sure! I just added that in!